module-apps/application-calendar/models/MonthModel.cpp

```diff
@@ -34,12 +34,7 @@ uint32_t MonthModel::getLastDay()
 
 uint32_t MonthModel::getFirstWeekOffset()
 {
-    if (this->firstWeekDayNumb == 0) {
-        return 6;
-    }
-    else {
-        return this->firstWeekDayNumb;
-    }
+    return this->firstWeekDayNumb;
```

Comment:
I know it's absolutely minor fix that doesn't impact the code functionality at all, but please do me a favor and change this variable name to firstWeekDayNumber - two additional characters and it's incomparably much more descriptive... :P

Reply:
Ok

Reply:
Done